src/Symfony/Component/Ldap/Entry.php

```diff
@@ -76,11 +89,11 @@ public function getAttributes()
      * Sets a value for the given attribute.
      *
      * @param string $name
-     * @param array  $value
+     * @param array $value
```

Comment:
must be reverted. We align argument names in the phpdoc

Reply:
this is done.